### DISCUSSION TITLE
Release v0.3.7

### DISCUSSION THREAD

**Discussion by alvin-reyes** [application-research/estuary#811] [Announcements]
## What's Changed
* Cache /viewer and auth instead of contents, zones by @neelvirdy in https://github.com/application-research/estuary/pull/791
* add logs and fix tracker by @en0ma in https://github.com/application-research/estuary/pull/792
* respect api queue engine by @en0ma in https://github.com/application-research/estuary/pull/793
* make sure non staged contents go in for deal making by @en0ma in https://github.com/application-research/estuary/pull/794
* add more logs and do not pin without content id by @en0ma in https://github.com/application-research/estuary/pull/795
* refactor shuttle connection by @en0ma in https://github.com/application-research/estuary/pull/797
* getStagedContentsGroupedByLocation improvement by @en0ma in https://github.com/application-research/estuary/pull/799
* added cli command to init shuttle by @franklovefrank in https://github.com/application-research/estuary/pull/760
* fix staging zone created by @en0ma in https://github.com/application-research/estuary/pull/809
* fix empty delegates and improve code readability by @en0ma in https://github.com/application-research/estuary/pull/810

## New Contributors
* @franklovefrank made their first contribution in https://github.com/application-research/estuary/pull/760

**Full Changelog**: https://github.com/application-research/estuary/compare/v0.3.6...v0.3.7

<hr /><em>This discussion was created from the release <a href='https://github.com/application-research/estuary/releases/tag/v0.3.7'>Release v0.3.7</a>.</em>

**Comment by WJSugar** [comment]
邮件已收到  谢谢
